docker/release.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+# Usage:
+# // To build and publish the Docker image to the internal container registry.
+# $ PUBLIC_DESTINATION=internal bash release.sh
+# // To publish the Docker image to the public container registry.
+# $ PUBLIC_DESTINATION=public bash release.sh
+set -ex
+
+INTERNAL_CONTAINER_REGISTRY='us.gcr.io/container-monitoring-storage/stackdriver-logging-agent'
```

Comment:
Do we really want this in the public repo? Would this not be embedded in the container build trigger?